Print_List_of_Odd_Numbers.py

```diff
@@ -18,6 +18,9 @@ def print_error_messages():
   if(n==0):
     start+=1
     print(start)
-  else:
-    if(n%2!=0):
-    print(start)
+    patch-2
```

Comment:
If possible use a function.
**WHY?**
That is the first step to good programming practices.
Function, that is the basic to writing to good programs.
Implement it.